Kernel/Heap/kmalloc.cpp

```diff
@@ -55,6 +55,7 @@ struct KmallocGlobalData {
 
     void add_subheap(u8* storage, size_t storage_size)
     {
+        static_assert(sizeof(KmallocSubheap) <= PAGE_SIZE);
```

Comment:
we usually have these directly under the definitions

Reply:
Right, but that would lose the context. The only reason we want the sub heap to be this size is this specific subtraction.